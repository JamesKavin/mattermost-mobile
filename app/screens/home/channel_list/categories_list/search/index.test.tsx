// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {renderWithIntlAndTheme} from '@test/intl-test-helper';

import Search from './index';

test('Search Field should match snapshot', () => {
    const {toJSON} = renderWithIntlAndTheme(
        <Search/>,
    );

    expect(toJSON()).toMatchSnapshot();
});
