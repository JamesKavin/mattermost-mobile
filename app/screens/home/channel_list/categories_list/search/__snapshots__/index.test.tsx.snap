// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Field should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(255,255,255,0.12)",
      "borderRadius": 8,
      "flex": 1,
      "flexDirection": "row",
      "height": 40,
      "justifyContent": "flex-start",
      "marginVertical": 20,
      "maxHeight": 40,
      "padding": 8,
      "width": "100%",
    }
  }
>
  <Icon
    name="magnify"
    style={
      Object {
        "color": "rgba(255,255,255,0.72)",
        "fontSize": 24,
        "width": 24,
      }
    }
  />
  <TextInput
    placeholder="Find Channels"
    placeholderTextColor="rgba(255,255,255,0.72)"
    style={
      Array [
        Array [
          Object {
            "fontFamily": "OpenSans",
            "fontSize": 16,
            "fontWeight": "400",
            "lineHeight": 24,
          },
          Object {
            "textAlignVertical": "center",
          },
        ],
        Object {
          "alignContent": "center",
          "alignItems": "center",
          "color": "#ffffff",
          "flex": 1,
          "height": 40,
          "marginLeft": 5,
          "marginTop": -2,
        },
      ]
    }
    testID="channel_list.search_field.find_channels.input"
  />
</View>
`;
